src/feat/pitch-functions.cc

```diff
@@ -1077,6 +1085,8 @@ void OnlinePitchFeatureImpl::AcceptWaveform(
   for (int32 frame = start_frame; frame < end_frame; frame++) {
     // start_sample is index into the whole wave, not just this part.
     int64 start_sample = static_cast<int64>(frame) * frame_shift;
+    // BP: handles snip_edges == false in a more graceful way
```

Comment:
Clease don't have comments like this.  Comments should explain what the code does-- not how the code has been changed, which is meaningless without seeing the history.   E.g., more graceful than what?